modules/virt-preventing-workload-updates-during-control-plane-only-update.adoc

```diff
@@ -206,4 +228,4 @@ $ oc get vmim -A
 
 .Next steps
 
-* You can now unpause the worker nodes' machine config pools.
\ No newline at end of file
+* You can now unpause the worker nodes' machine config pools.
```

Comment:
```suggestion
* Unpause the machine config pools for each compute node.
```

See https://redhat-documentation.github.io/supplementary-style-guide/#compute-node.

Reply:
Is the next steps related to the "Configuring workload update methods" section? If not, maybe hyperlink the sentence?

Reply:
Fixed as suggested, thank you. This is in a part that I did not review but the entire assembly has problems. Rather than trying to hyperlink things here, I'd really like to take a look at the whole assembly after the release and ensure that the information flows logically.